templates/triton_services/triton_app.py

```diff
@@ -51,7 +50,7 @@ def generate(self, prompt: str) -> StreamingResponse:
             buffer = io.BytesIO()
```

Comment:
converting to an `Image` type and then writing it into the buffer seems pointless here -- you already have a raw byte array above, why not just return that directly? does `fromarray` do any actual transformation of the data?

Reply:
what does it do exactly? numpy array of uint8 is already a byte buffer

Reply:
Talked offline, this seems to be official/ easiest way to converting RGB array into JPEG image bytes 🙂